packages/dart_frog/lib/src/request.dart

```diff
@@ -158,3 +154,5 @@ class Request {
     return Request._(_request.change(headers: headers, path: path, body: body));
   }
 }
+
+final _bodyFutureExpando = Expando<Future<String>>('dart_frog.request.body');
```

Comment:
Not that familiar with Expandos, would love to see a description on the PR to explain how this improves performance

Reply:
You all would have to measure it. 

On the VM, they're pretty cheap. And now you're avoiding copying everything all the time which should be a lot cheaper